src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1669,10 +1669,10 @@ type Plan {
     nextDateTime: Long @deprecated(reason: "Use nextPageCursor instead")
 
     """
-    This is the time window used by the raptor search. The window is an optional parameter and
-    OTP might override it/dynamically assign a new value. The unit is seconds.
+    This is the time window used by the raptor search. The input searchWindow is an optional
```

Comment:
Should we comment here that the input searchWindow gets overridden when searching with the pageCursor?